Source/Core/Common/Timer.cpp

```diff
@@ -254,7 +254,7 @@ std::string Timer::GetDateTimeFormatted(double time)
 
 #ifdef _WIN32
   wchar_t tmp[32] = {};
-  wcsftime(tmp, sizeof(tmp), L"%x %X", localTime);
+  wcsftime(tmp, _countof(tmp), L"%x %X", localTime);
```

Comment:
I realize this is Windows specific code, but is there a reason why we cannot use [`std::size`](https://en.cppreference.com/w/cpp/iterator/size) for this?

Reply:
Thanks, I tried converting the `_countof` instances in this commit to `std::size`, but it then triggers narrowing warnings, as the windows apis are using `int` and `DWORD`, whereas `std::size` returns `size_t`. Of course the warnings are harmless, but I'd rather just use the method traditional to the platform here).

Reply:
I updated this instance (as param is `size_t` here), tho :)